Dockerfile.nidx

```diff
@@ -0,0 +1,13 @@
+#
+# Dockerfile for nucliadb's index node, compiling nidx in the docker build.
+# For a version that uses pre-built binaries (outside Docker) see `Dockerfile.nidx_prebuild`
+#
+
+FROM rust:1.82.0-bookworm AS builder
+COPY nucliadb_protos/*proto /app/nucliadb_protos/
+COPY nidx /app/nidx
+RUN apt update && apt install -y protobuf-compiler
```

Comment:
Moving this line above the `COPY` would be cached better

Reply:
Oh yeah, my mistake! Thanks!